server/conf.go

```diff
@@ -587,34 +653,46 @@ func (conf *Configuration) verifySdJwtIssuanceSettings() error {
 		return nil
 	}
 
-	privKeyBytes, err := common.ReadKey(sdConf.JwtPrivateKey, sdConf.JwtPrivateKeyFile)
-	if err != nil {
-		return fmt.Errorf("failed to read private key from file: %v", err)
-	}
+	privKeys, err := readSdJwtIssuerPivKeys(conf.SdJwtIssuanceSettings.SdJwtIssuerPrivKeysPath)
 
-	sdConf.JwtEcdsaPrivateKey, err = sdjwtvc.DecodeEcdsaPrivateKey(privKeyBytes)
 	if err != nil {
-		return errors.WrapPrefix(err, "failed to parse SD-JWT VC private key", 0)
+		return fmt.Errorf("failed to read sdjwt issuer private keys: %v", err)
 	}
 
-	certBytes, err := common.ReadKey(sdConf.IssuerCertificateChain, sdConf.IssuerCertificateChainFile)
-	if err != nil {
-		return fmt.Errorf("failed to obtain SD-JWT VC issuer certificate chain: %v", err)
-	}
+	certChains, err := readSdJwtIssuerCertChains(conf.SdJwtIssuanceSettings.SdJwtIssuerCertificatesPath)
 
-	certChain, err := utils.ParsePemCertificateChainToX5cFormat(certBytes)
 	if err != nil {
-		return fmt.Errorf("failed to parse x.509 certificate chain: %v", err)
-	}
-	if len(certChain) == 0 {
-		return fmt.Errorf("SD-JWT VC x.509 issuer certificate chain is empty")
+		return fmt.Errorf("failed to read sdjwt issuer certificate chains: %v", err)
 	}
 
-	if sdConf.Issuer == "" {
-		return errors.New("SD-JWT VC issuance enabled, but no issuer specified in configuration")
+	if len(privKeys) != len(certChains) {
+		return fmt.Errorf("the number of sdjwtvc cert chains and private keys don't match")
 	}
 
-	sdConf.X509CertChain = certChain
+	sdConf.Issuers = map[irma.IssuerIdentifier]*SdJwtIssuer{}
+	for issuerId, privKey := range privKeys {
+		certChain, ok := certChains[issuerId]
+		if !ok {
+			return fmt.Errorf("expected cert chain for %v, but it's not present", issuerId)
+		}
+
+		x5cChain, err := utils.ConvertPemCertificateChainToX5cFormat(certChain)
+		if err != nil {
+			return fmt.Errorf("failed to convert cert chain for %v to x5c format: %v", issuerId, err)
+		}
+
+		issuerUrl, err := utils.ObtainIssuerUrlFromCertChain(certChain)
```

Comment:
Maybe good to log which `issuerUrl` is chosen to be configured, as certificates allow for multiple SAN-URI's.